lib/stat_tracker.rb

```diff
@@ -144,4 +145,22 @@ def worst_offense
     league_stat.worst_offense
   end
 
+  def percentage_home_wins
+    game_collection = GameCollection.new(@games_file)
+    game_collection.create_pct_data
+    game_collection.percentage_home_wins
+  end
+
+  def percentage_visitor_wins
+    game_collection = GameCollection.new(@games_file)
+    game_collection.create_pct_data
+    game_collection.percentage_visitor_wins
+  end
+
+  def percentage_ties
+    game_collection = GameCollection.new(@games_file)
+    game_collection.create_pct_data
+    game_collection.percentage_ties
+  end
```

Comment:
game_collection.create_pct_data should probably happen IN the file but we can save that for the spare refactoring time that probably won't exist. so it's not ideal but it IS functional, so i'm ok with it for now.

Reply:
I totally agree, we will address that in a later refactor